src/components/galaciuc.jsx

```diff
@@ -2,22 +2,182 @@
 
 import React from "react";
 
+import { Row, Col, Grid } from "react-bootstrap";
+import Header from "./header";
+
+import "./galaciuc.less"
+
 
 export default React.createClass({
   displayName: "GalaciucPage",
 
   render() {
-    return <div>
-      <h1>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h1>
-      <h2>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h2>
-      <h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h3>
-      <h4>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h4>
-      <h5>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h5>
-      <h6>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h6>
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-      <p><a href="#" className="link link-primary">Link Primary</a></p>
-      <p><a href="#" className="link link-secondary">Link Secondary</a></p>
-      <p><a href="#" className="link link-ternary">Link Ternary</a></p>
+    return <div className="galaciuc">
+      <div className="blue-section-wrapper">
+        <Grid className="blue-section">
+          <Row>
+            <Col xs={12}>
+              <Header isLoggedIn={this.props.isLoggedIn}
+                      login={this.props.login}
+                      logout={this.props.logout} />
+            </Col>
+          </Row>
+          <Row>
+            <Col xs={12}>
+              <h1>Despre Concursul InfoEducație</h1>
+              <h2>istoric și regulament</h2>
+            </Col>
+          </Row>
+          <Row className="big-spacing" />
+        </Grid>
+      </div>
+      <Grid className="white-section">
+        <Row>
+          <Col mdOffset={1} md={10} xsOffset={1}>
```

Comment:
Fără xsOffset. Nu ți-a intrat alinierea pentru că ai p-uri in row fara col. Sterge row de la ele. Am descoperit ca poti sa ai small-spacing apoi p amestecate, fara sa fie doar row-uri.